### ISSUE TITLE
[Feature Request]: Median, IQR, MAD etc. for non-parametric ANOVAs & paired tTest

### ISSUE THREAD

**Issue by DrMarkGS** [jasp-stats/jasp-issues#1062]
Hi folks
When carrying out non-parametric analyses students are told to report the descriptives as median and IQR's or MAD.  However the default descriptives in t- tests, ANOVA etc are always mean, SD or SE.

Is there any possibility that if non-parametric analyses are selected that the descriptive show medians etc.  This will stop students naturally presenting the default means & SD which are inappropriate.

Just a cunning plan
Mark

**Comment by boutinb** [comment]
@EJWagenmakers ?

**Comment by EJWagenmakers** [comment]
Makes perfect sense, we should do this. I'll assign this to @JohnnyDoorn for now (to do after he finishes his thesis, obviously)
E.J.

**Comment by JohnnyDoorn** [comment]
Yes, it would be great to have the descriptives of the ranktransformed data!

**Comment by tomtomme** [comment]
Will partly be fixed by: https://github.com/jasp-stats/jaspTTests/pull/236
